### PR TITLE
GEN-1462 | Parse optional shop session create partner params from URL

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Parse URL and extract optional `shopSessionCreatePartner` params

## Justify why they are needed

- We currently don't accept "external member ID" but the plan is to do so in the future

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3583** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3583?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3584** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3584?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3586** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3586?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3587** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3587?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3591** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3591?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #3592** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3592?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #3593** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3593?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3593?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/features/widget/usePublishWidgetInitEvent.ts [hunk 1]
```diff
@@ -2,20 +2,20 @@ import { useRouter } from 'next/router'
 import { useEffect } from 'react'
 import { publishWidgetEvent } from './publishWidgetEvent'
 
-export const PARTNER_REQUEST_ID_SEARCH_PARAM = 'partnerRequestId'
+export const EXTERNAL_REQUEST_ID_SEARCH_PARAM = 'requestId'
```
Comment (guilhermespopolin) [inline apps/store/src/features/widget/usePublishWidgetInitEvent.ts:5]: Wonder if we should share a single constant here and in `parseSearchParams` 🤔

**Review by guilhermespopolin — APPROVED** [review]
Nice

**Comment by robinandeer** [comment]
### Merge activity

* **Nov 21, 1:29 PM**: @robinandeer started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3593).

**Pull request merged**